.dev/hooks/pre-commit.sysimage

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env julia -J.git/hooks/JuliaFormatterSysimage.so
```

Comment:
```suggestion
#!/usr/bin/env -S julia -J.git/hooks/JuliaFormatterSysimage.so
```

The `-S` flag should be used when passing arguments.

Reply:
Thanks.